heron/tools/cli/src/python/main.py

```diff
@@ -330,11 +331,15 @@ def execute(handlers, local_commands):
 
   command_line_args = vars(args)
 
-  # command to be execute
+  # set log level
+  log.set_logging_level(command_line_args)
+  Log.debug("Input Command Line Args: %s", command_line_args)
+
+# command to be execute
   command = command_line_args['subcommand']
   is_local_command = command in local_commands
 
-  if command == 'version':
+  if command == 'version' or command == 'standalone':
```

Comment:
suggest to put the standalone command to the local command list, instead of here as an or operation. It is  more complicated than the version command.

See below in main() function,
local_commands = ('help', 'version', 'config')